orchagent/intfsorch.cpp

```diff
@@ -177,6 +178,11 @@ bool IntfsOrch::addRouterIntfs(Port &port)
         return true;
     }
 
+    if (port.m_type == Port::PHY)
```

Comment:
Before removing bridge port, the vlan member for that {bridge port, default vlan} should also be deleted